### PR TITLE
💄Feature/8/quality content group

### PR DESCRIPTION
**Pull request by ashby-git**
**What did you do?**
- I created a component that uses 3 props to hold different information

**Screenshot**
Screen Shot 2021-09-06 at 18 05 49

**How to Check Feature/Bug Fix**
```
import './App.css';
import QualityContentGroup from './components/quality-page/quality-content-group.component';

function App() {
  return (
    <div className="container">
      <div className='background'>
        <div>
        </div>
      </div>
      <QualityContentGroup
        number="01"
        subheading={"100%オーガニックの\nココナッツのみを使用"}
        textbox="DEAR COCOのコスメに使用するココナッツは、化学肥料や農薬は一切不使用。太陽の光をいっぱいに浴びて育った100%オーガニックココナッツオイルを自社契約農家から直接仕入れ・加工しています。"
      />
    </div>
  );
}

export default App;
```

**Link to the Design**
(https://xd.adobe.com/view/c0f0ee74-781d-434f-9960-3ff85a002849-95a3/specs/?fbclid=IwAR3bcz-BSXJGrgTXDSVBv0y-PYnSKaPM075W3UEhqyWRb_iHMHv7nu77Hq4)

### CODE REVIEW

File: src/components/quality-page/quality-content-group.styles.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.line {
+    border-top: 2px solid #BFB4AD;
+    width: 31vw;
+    margin-top: 1vw;
+    margin-bottom: 1.5vw;
+}
+
+.textbox {
```
Comment (DanArcherOnline) [inline src/components/quality-page/quality-content-group.styles.scss:30]: textbox is super generic so it should be placed in app.css and used for all textboxes.
or make the name specific to what its addressing

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
- fix varying line space between subheading lines

File: src/components/quality-page/quality-content-group.styles.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    color: #BFB4AD;
+}
+
+.subheading {
+    font-size: 1.75em;
+    font-family: YuMincho;
+    font-style: italic;
+    color: #5A4537;
+    width: 22vw;
+    line-height: 6.6vh;
```
Comment (DanArcherOnline) [inline src/components/quality-page/quality-content-group.styles.scss]: line height shouldnt use vh.
its probably best to almost never use vh/vw with text, unless its purely decorative.
if you change the height of the browser it squishes together and lines may even overlap.
you can use px/em/rem for line height.

in fact, in all cases, you should try not to use vh/vw unless you see no other alternative.

**Review by DanArcherOnline — CHANGES_REQUESTED** [review]
also try updating the code in `app.js` to format the subheading correctly.
as in, make it so it starts a new line after "の".
message me if you get stuck

File: src/components/quality-page/quality-content-group.styles.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.top-section {
+    display: flex;
+}
+
```
[the remark this answers is not in the document]
Comment (ashby-git) [reply src/components/quality-page/quality-content-group.styles.scss:9]: Made these changes

File: src/components/quality-page/quality-content-group.styles.scss (hunk 1 above)
Comment (ashby-git) [reply src/components/quality-page/quality-content-group.styles.scss]: Changed the textbox to a more specific name

File: src/components/quality-page/quality-content-group.styles.scss (hunk 2 above)
Comment (ashby-git) [reply src/components/quality-page/quality-content-group.styles.scss]: That was just a mistake I forgot to change last time, changed to line height.

**Comment by ashby-git** [comment]
Updated the app.js code example in original post

**Pull request merged**